app/services/search/feed_content.rb

```diff
@@ -23,11 +23,30 @@ def search_documents(params:)
       def prepare_doc(hit)
         source = hit.dig("_source")
         source["tag_list"] = hit["tags"]&.map { |t| t["name"] } || []
+        source["id"] = hit.dig("_source", "id").split("_").last.to_i
+        source["tag_list"] = hit.dig("_source", "tags")&.map { |t| t["name"] } || []
+        source["flare_tag"] = hit.dig("_source", "flare_tag_hash")
         source["user_id"] = hit.dig("_source", "user", "id")
         source["highlight"] = hit["highlight"]
+        source["readable_publish_date"] = hit.dig("_source", "readable_publish_date_string")
+
+        source.merge!(timestamps_hash(hit))
```

Comment:
do we need to merge in place? maybe we can just return `source.merge(timestamp_hash(hit))` from the method

Reply:
Oh, yeah we can totally do that, this was a couple of lines up when I originally pulled it out to a separate method which is where the in-place came from.